Simplified/NYPLBookAcquisitionPath.h

```diff
@@ -12,6 +12,8 @@ static NSString * const _Nonnull ContentTypeFindaway = @"application/vnd.library
 static NSString * const _Nonnull ContentTypeOpenAccessAudiobook = @"application/audiobook+json";
 static NSString * const _Nonnull ContentTypeOpenAccessPDF = @"application/pdf";
 static NSString * const _Nonnull ContentTypeFeedbooksAudiobook = @"application/audiobook+json;profile=\"http://www.feedbooks.com/audiobooks/access-restriction\"";
+static NSString * const _Nonnull ContentTypeOverdriveCatalog = @"application/atom+xml;type=entry;profile=opds-catalog";
```

Comment:
I'm really sorry I didn't catch this sooner, I apologize for the back and forth. But we should avoid putting static definitions in a header file, because header files can be included anywhere, and therefore you will continue to _define_ identically named symbols wherever you include the file.

The right way is to move these to the .m if they are only used there. If they used somewhere else, as I suspect, then you should declare them as (e.g.)
```
extern NSString * const _Nonnull ContentTypeOverdriveCatalog;
```
and in the .m:
```
NSString * const _Nonnull ContentTypeOverdriveCatalog = @"application/atom+xml;type=entry;profile=opds-catalog";
```